tsdb/functions_test.go

```diff
@@ -529,17 +529,17 @@ func TestMapTop(t *testing.T) {
 				values: []testPoint{
 					{"", 20, int64(99), map[string]string{"host": "a"}},
 					{"", 10, int64(53), map[string]string{"host": "a"}},
-					{"", 10, int64(99), map[string]string{"host": "a"}},
+					{"", 10, int64(99), map[string]string{"host": "b"}},
 				},
 			},
 			exp: positionOut{
 				callArgs: []string{"host"},
 				points: PositionPoints{
-					PositionPoint{10, int64(99), map[string]string{"host": "a"}},
+					PositionPoint{10, int64(99), map[string]string{"host": "b"}},
```

Comment:
I'm not sure I understand this change.  The test was testing that it would pick max values, and then sort by time next to be deterministic.  The tags were left identical to validate that under this scenario it had could only resort to time for sorting.

If you want to test that it sorts by tags as a tertiary sort after value and time, I think we should add another test and leave this one as is.